utils.go

```diff
@@ -3,18 +3,17 @@ package redisgraph
 import (
 	"crypto/rand"
 	"fmt"
-	"reflect"
 	"strings"
 )
 
 // go array to string is [1 2 3] for [1, 2, 3] array
 // cypher expects comma separated array
 func arrayToString(arr interface{}) interface{} {
-	v := reflect.ValueOf(arr)
-	var arrayLength = v.Len()
+	v := arr.([]interface{})
+	var arrayLength = len(v)
 	strArray := []string{}
 	for i := 0; i < arrayLength; i++ {
-		strArray = append(strArray, fmt.Sprintf("%v", ToString(v.Index(i))))
+		strArray = append(strArray, fmt.Sprintf("%v", ToString(v[i])))
```

Comment:
I think what we want here is to wrap `v[i]` in quotes if `v[i]` is of type string,
if this is indeed the case, use a different function, e.g. `QuoteString(v[i])`

Reply:
The `ToString` method does it